addons/product/product.py

```diff
@@ -942,6 +942,14 @@ def _select_seller(self, cr, uid, product_id, partner_id=False, quantity=0.0, da
             break
         return res
 
+    def _get_items(self, cr, uid, ids, field_name, args, context=None):
+        res = {}
+        for prod_id in ids:
```

Comment:
why not `for prod in self.browse(cr, uid, ids, context):`? You don't like the cache and prefetching?